tests/element_test_vectors.rs

```diff
@@ -645,8 +622,20 @@ mod non_blocking_native_element_tests {
             &[]
         }
 
-        fn make_reader() -> Self::ReaderApi {
-            NonBlockingNativeElementReader
+        fn make_reader(data: &[u8]) -> IonResult<Reader<'_>> {
```

Comment:
🗺️ This method needs to construct a format-agnostic non-blocking reader, which isn't currently possible with the existing `ReaderBuilder`. Fixing that is being tracked in #484.

Reply:
Is that worth leaving a comment here so that #484 cleanup can easily find all the relevant pieces of code?